sklearn/pipeline.py

```diff
@@ -638,6 +642,10 @@ def _get_name(name, est):
                             name_details=name_details,
                             dash_wrapped=False)
 
+    def _more_tags(self):
+        print(self.steps[-1][1])
+        return self.steps[-1][1]._get_tags()
```

Comment:
This may not be true because the first step can handle `nans` where the last step may not be able to handle `nan`s.

Reply:
You are absolutely right. This was one of the main doubts, to use the tags of the last estimator or to take the tags for all estimators in the `Pipeline`.